NetworkPackageDemo/NetworkPackageDemo/Sources/NetworkPackage/RequestBuilder.swift

```diff
@@ -10,11 +10,11 @@ import Foundation
 
 public protocol RequestBuilder {
     var baseURL: String { get }
-
     var method: HttpMethod { get }
     var path: String { get }
     var parameters: [String: Any] { get }
     var headers: [String: String] { get }
+    var bodyParamters: [String: Any] {get}
```

Comment:
remove this, and  use parameters var to do the same job

Reply:
Done